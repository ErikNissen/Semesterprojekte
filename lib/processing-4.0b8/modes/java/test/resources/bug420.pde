version https://git-lfs.github.com/spec/v1
oid sha256:700b95bb5dffcd6984d63d0c4ccfc6d49c225c3f679512865dbfc84407708010
size 63
