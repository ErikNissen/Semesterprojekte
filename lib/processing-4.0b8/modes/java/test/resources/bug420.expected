version https://git-lfs.github.com/spec/v1
oid sha256:1315ee43c9eca44cba290d0ed229430ef5d235a6ee56d9a855dccf1b4400a485
size 71
